nvm.sh

```diff
@@ -721,26 +797,26 @@ nvm_print_formatted_alias() {
   if [ -z "${NVM_NO_COLORS}" ] && nvm_has_colors; then
     ARROW='\033[0;90m->\033[0m'
     if [ "_${DEFAULT}" = '_true' ]; then
-      NEWLINE=' \033[0;37m(default)\033[0m\n'
+      NEWLINE=" \033[${DEFAULT_COLOR}(default)\033[0m\n"
```

Comment:
if this is a global env var, it could easily conflict with a user's own env vars, since nvm is sourced inside the user's shell session. all env vars need to be prefixed with `NVM_` to avoid that; additionally, i'd strongly prefer avoiding storing global state in multiple env vars.

Reply:
Added `NVM_` prefix.